opnode/flags/p2p_flags.go

```diff
@@ -192,6 +192,14 @@ var (
 		Value:     "opnode_discovery_db",
 		EnvVar:    p2pEnv("DISCOVERY_PATH"),
 	}
+	SequencerP2PKeyFlag = cli.StringFlag{
```

Comment:
I know that geth uses this same scheme, do you think we could use an encrypted keystore as well so that we can fully migrate to only using encrypted keystores?

Reply:
The "sequencer p2p key" is the key used to sign application layer data, and should use an encrypted keystore, like the batchsubmitter and output submitter also should.

The "p2p node key" (different!) used for the node identity is much less critical, and every user has one. I think it makes sense to follow the geth (and eth2) pattern there, to generate one if it doesn't exist, and persist it as file to disk.